tests/image/affine_transformations_test.py

```diff
@@ -39,8 +39,8 @@ def test_matrix_center():
         [0, 0],
     ]), -1)
     x_rotated90 = np.expand_dims(np.array([
+        [1, 0],
```

Comment:
The rotation should be clock-wise I think.

Reply:
It happened so that I re-wrote the whole module yesterday and spent some time figuring out exact directions and quantities of each transform. I concluded that the rotation should be CCW, which led me to finding this bug. 

The reasoning is as follows:
1. The Y axis in images points down, hence the positive rotation should be CW. Consequently, the negative direction is CCW.
2. Affine transform used here is the inverse (aka backward, pull, etc.)  transform (see [affine_transform](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.affine_transform.html)). Therefore, it "negates" its arguments. Hence, for positive angle, the rotation is negative, namely, CCW. 
Similarly, for positive translation, the image moves in the negative direction. 

P. S.
As an additional sanity check,  I verified that the behavior here matches [scikit-image](https://scikit-image.org/docs/dev/api/skimage.transform.html#skimage.transform.AffineTransform)